iocore/net/UnixNet.cc

```diff
@@ -387,9 +375,9 @@ NetHandler::process_ready_list()
     // Initialize the thread-local continuation flags
     set_cont_flags(vc->control_flags);
     if (vc->closed)
-      close_UnixNetVConnection(vc, trigger_event->ethread);
+      close_UnixNetVConnection(vc, this->thread);
```

Comment:
I'm concerned about the change here - I suspect this PR should be updated to reflect the old usage. Please check with @shinrich on that, or do a `git blame` to see who and what commit changed lines 390 and 392 to use the different thread. It possible though it's part of the change from event driven to event loop driven (which would remove `trigger_event`). Needs to be checked.

Reply:
Please create a new pr for replace `trigger_event->ethread` with `this->thread`.

Reply:
On reflection, I realized this change was implicit in the PR because `trigger_event` no longer exists. The mutex should be the same mutex.

Reply:
The `UnixNetVConnection::reenable(VIO *vio)` still access the `trigger_event`.